includes/block-patterns/class-sensei-block-patterns.php

```diff
@@ -61,6 +62,44 @@ public function register_block_patterns_category() {
 		);
 	}
 
+	/**
+	 * Temporary pattern for course list block because default patterns changes the Post Type to 'post' instead of 'course'.
+	 *
+	 * @access private
+	 */
+	public function register_course_list_block_pattern() {
+		register_block_pattern(
+			'dummy-course-list-query',
+			[
+				'title'      => __( 'Grid of courses', 'sensei-lms' ),
+				'categories' => array( 'query' ),
+				'blockTypes' => array( 'core/query' ),
+				'content'    => '<!-- wp:query {"query":{"offset":0,"postType":"course","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","sticky":"","perPage":4},"displayLayout":{"type":"flex","columns":3},"layout":{"inherit":false}} -->
```

Comment:
this should be a separate file, but I guess it's ok since this is just a dummy for the beginning